devbox_test.go

```diff
@@ -68,9 +68,21 @@ func assertPlansMatch(t *testing.T, expected *plansdk.Plan, actual *plansdk.Plan
 	// Check that input files are the same for all stages.
 	// Depending on where the test command is invoked, the input file paths can be different.
 	// We will compare the file name only.
-	assert.ElementsMatch(expected.InstallStage.GetInputFiles(), getFileNames(actual.InstallStage.GetInputFiles()), "InstallStage.InputFiles should match")
-	assert.ElementsMatch(expected.BuildStage.GetInputFiles(), getFileNames(actual.BuildStage.GetInputFiles()), "BuildStage.InputFiles should match")
-	assert.ElementsMatch(expected.StartStage.GetInputFiles(), getFileNames(actual.StartStage.GetInputFiles()), "StartStage.InputFiles should match")
+	assert.ElementsMatch(
+		expected.InstallStage.GetInputFiles(),
+		getFileNames(actual.InstallStage.GetInputFiles()),
+		"InstallStage.InputFiles should match",
+	)
+	assert.ElementsMatch(
+		expected.BuildStage.GetInputFiles(),
+		getFileNames(actual.BuildStage.GetInputFiles()),
+		"BuildStage.InputFiles should match",
+	)
+	assert.ElementsMatch(
+		expected.StartStage.GetInputFiles(),
+		actual.StartStage.GetInputFiles(),
+		"StartStage.InputFiles should match",
+	)
```

Comment:
verified with @LucilleH that it makes sense to drop `getFileNames` for StartStage.

Because:
1. The motivation to introduce `getFileNames` was to normalize the filepaths used. The filepaths would differ based on which directory the `go test` command is invoked from.
2. This doesn't apply to StartStage, since the InputFiles are relative to the BuildStage's container. And _not_ relative to the local laptop/CICD invoking `go test`.